flake8_trio/visitors/visitors.py

```diff
@@ -127,35 +128,38 @@ def visit_Call(self, node: ast.Call):
             and isinstance(node.func, ast.Attribute)
             and isinstance(node.func.value, ast.Name)
             and (
-                (node.func.value.id == "trio" and node.func.attr in trio_async_funcs)
+                (
+                    node.func.value.id in ("trio", "anyio")
+                    and node.func.attr in trio_async_funcs
```

Comment:
Some of `trio_async_funcs` don't exist in `anyio`, and vice-versa; we should probably make a second list (well, tuple) for `anyio` and check them separately.

Reply:
oh hum, it also doesn't have `.open_nursery()`